TimberAPI/Dependency/Dependency.cs

```diff
@@ -35,39 +37,32 @@ public void AddConfigurator(IConfigurator configurator, SceneEntryPoint entryPoi
             }
         }
 
-        /**
-         * Inject configurators into MasterScene (InGame)
-         */
+        
+        // Inject configurators into MasterScene (InGame)
```

Comment:
How about renaming the method to `InjectIntoMasterScene`? That would allow removing the comment entirely, because the signature would be self-explanatory.

If you want to keep the comment: shouldn't it say "Inject a container definition..."?